ush/ufsda/yamltools.py

```diff
@@ -206,8 +206,8 @@ def calc_time_vars(config):
     bkg_time_obj = valid_time_obj
     config['BKG_YYYYmmddHHMMSS'] = bkg_time_obj.strftime('%Y%m%d.%H%M%S')
     config['BKG_ISOTIME'] = bkg_time_obj.strftime('%Y-%m-%dT%H:%M:%SZ')
-    config['LAND_BKG_YYYYmmddHHMMSS'] = bkg_time_obj.strftime('%Y%m%d.%H%M%S')
-    config['LAND_BKG_ISOTIME'] = bkg_time_obj.strftime('%Y-%m-%dT%H:%M:%SZ')
+    config['snow_BKG_YYYYmmddHHMMSS'] = bkg_time_obj.strftime('%Y%m%d.%H%M%S')
+    config['snow_BKG_ISOTIME'] = bkg_time_obj.strftime('%Y-%m-%dT%H:%M:%SZ')
```

Comment:
snow_BKG_YYYYmmddHHMMSS -> SNOW_BKG_YYYYmmddHHMMSS, as LAND is capital, I am not sure. The same case is for snow_BKG_ISOTIME -> SNOW_BKG_ISOTIME.

Reply:
Others looks fine with me.

Reply:
Thanks @YoulongXia-NOAA for catching my mistake!